src/read.c

```diff
@@ -2386,7 +2410,7 @@ static avifBool avifParseItemInfoEntry(avifMeta * meta, const uint8_t * raw, siz
 
     avifDecoderItem * item = avifMetaFindItem(meta, itemID);
     if (!item) {
-        avifDiagnosticsPrintf(diag, "Box[infe] has an invalid item ID [%u]", itemID);
+        avifDiagnosticsPrintf(diag, "%s: Box[infe] with item_type %.4s has an invalid item ID [%u]", s.diagContext, itemType, itemID);
```

Comment:
Nit: Whether an underscore '_' is used or not used in "item_type" or "item ID" could be consistent.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/1502